assembly/runtime/util.ts

```diff
@@ -68,7 +73,8 @@ export namespace util {
   }
 
   /**
-   * Reads contents of a register into a Uint8Array.
+   * Reads contents of a register into a Uint8Array. Note: this is a low level
```

Comment:
Maybe move the `Note` to the next line and maybe this should be listed with `@package` since end users will be able to use it, but it's mainly meant for internal package use.